06_Interacting_with_Lean.org

```diff
@@ -495,7 +500,7 @@ with the extension =.olean=, for "object Lean".
 
 Assuming the directory that contains Lean is in your system path, you
 can run lean on a file =foo.lean= from a system command line by typing
-=lean foo=. If =foo= imports other files, by default Lean looks for
+=lean foo.lean=. If =foo= imports other files, by default Lean looks for
```

Comment:
I'm a bit confused about this one, did it work without the file extension at some point?